srcpkgs/klfc/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'klfc'
+pkgname='klfc'
+short_desc='Keyboard layout files creator'
+maintainer='Nadeŭka <nadevko@riseup.net>'
+license='GPL-3.0-or-later'
+homepage='https://github.com/39aldo39/klfc'
+revision='1'
+
+version='1.5.7'
+checksum='5a052c6f59917e99593e507e460e37b6633a388a43ef9ed0401b297c723c1eba'
+
+build_style='haskell-stack'
+stackage='nightly-2021-10-07'
+distfiles="https://github.com/39aldo39/klfc/archive/refs/tags/v$version.tar.gz"
+nopie_files='/usr/bin/klfc'
+python_version='3'
```

Comment:
remove the blank lines in this section and run xlint

Reply:
I made all the amends you asked for.